compiler/main.bal

```diff
@@ -5,17 +5,25 @@ import wso2/nballerina.err;
 
 import ballerina/io;
 import ballerina/file;
+import ballerina/regex;
```

Comment:
we are trying not to use too many library functions, so we can bootstrap with less work.
Better to avoid regex.

Reply:
Right. Definitely no regex.